### PR TITLE
#85 - Provide support for custom drush folders.

### PR DESCRIPTION
**Pull request by fluxsauce**
https://github.com/aquifer/aquifer/issues/85

Testing:

``` bash
cd /var/tmp
node ~/projects/aquifer/bin/aquifer.js create drushtest
cd drushtest
node ~/projects/aquifer/bin/aquifer.js build
ls -la build/sites/all/drush
```

### CODE REVIEW

File: src/aquifer.default.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "settings": "settings",
     "build": "build",
     "root": "root",
+    "drush": "drush",
```
Comment (patrickocoffeyo) [inline src/aquifer.default.json:9]: Should this be a default? Will the majority of sites using Aquifer need to use drushrc.php?

Comment (patrickocoffeyo) [inline src/aquifer.default.json]: Should this be a default?

Comment (fluxsauce) [inline src/aquifer.default.json:9]: I feel it should be, it creates an empty placeholder. Very common need and I'd consider it a best practice.

Comment (patrickocoffeyo) [inline src/aquifer.default.json:9]: Sounds good!

**Comment by patrickocoffeyo** [comment]
:shipit:

**Pull request merged**